drivers/sensor/bosch/bme680/bme680.c

```diff
@@ -444,12 +438,43 @@ static int bme680_init(const struct device *dev)
 		return err;
 	}
 
-	err = bme680_reg_write(dev, BME680_REG_CTRL_MEAS,
-			       BME680_CTRL_MEAS_VAL);
+	return bme680_reg_write(dev, BME680_REG_CTRL_MEAS, BME680_CTRL_MEAS_VAL);
```

Comment:
Shouldn't we get rid of the `return err;` below?

Reply:
fixed